service/history/api/queryworkflow/api.go

```diff
@@ -181,6 +182,7 @@ func Invoke(
 				rawMatchingClient,
 				matchingClient,
 				scope,
+				mutableState.GetExecutionInfo().Priority,
```

Comment:
This is after `workflowLease.GetReleaseFn()(nil)`.. I don't think we can access `mutableState.GetExecutionInfo()` here?

Reply:
I wish you could use emojis in Go function names; I didn't notice the `GetReleaseFn` line there at all.